common/app/views/fragments/bashers/ThisIsTheNHS.scala.html

```diff
@@ -0,0 +1,41 @@
+@()(implicit request: RequestHeader)
+
+<div class="gs-container">
+    <div class="this-is-the-nhs">
+        <h1 class="title title--first">this is</h1>
+        <h1 class="title title--second">the NHS</h1>
+        <p class="description">The story of one of the most complex organisations in the world, told through the voices of those on the frontline.</p>
+
+        <div class="stripe stripe--dark-blue"></div>
+        <div class="stripe stripe--blue"></div>
+        <div class="stripe stripe--white"></div>
+
+        <div class="line">
+            <svg class="line__image" width="2200" height="419.084" viewBox="0 0 2200 419.084"><path clip-rule="evenodd" stroke="#FDBA12" stroke-width="12" stroke-miterlimit="10" d="M0 287.004h778l14.2-47.5 25.2 108 27.2-161 22.2 197 18.2-96.5h433l20.4-84.5 20.4 142.447 20.4-190.446 20.4 216 20.4-83.5h312l27.4-140.5 26.6 195.5 28-288 34 269 21-36h327" fill="none"/></svg>
+        </div>
+
+        <div class="circle"></div>
+        <div class="circle circle--lower"></div>
+
+        <img class="hand hand--left" src="http://static.guim.co.uk/sys-images/Guardian/Pix/pictures/2016/1/7/1452167267487/handleftcompressor.png" />
```

Comment:
Yeah, there is `Configuration.static.path` please use this as it will use the secure version if the user is on HTTPS.